templates/github/.github/workflows/scripts/install.sh.j2

```diff
@@ -17,6 +17,7 @@ if [[ "$TEST" = "docs" || "$TEST" = "publish" ]]; then
   pip install -r ../pulpcore/doc_requirements.txt
 {%- endif %}
   pip install -r doc_requirements.txt
+  pip install "Jinja2<3.1"
```

Comment:
Should this go into doc_requirements?

Reply:
https://github.com/pulp/pulpcore/pull/2429